Makefile.js

```diff
@@ -133,30 +132,30 @@ function release(type) {
     exec("git push origin master --tags");
 
     // now push the changelog...changes to the tag
-    // echo("Publishing changes to github release");
+    echo("Publishing changes to github release");
     // this requires a github API token in process.env.ESLINT_GITHUB_TOKEN
     // it will continue with an error message logged if not set
-    // ghGot("repos/eslint/eslint/releases", {
-    //     body: {
-    //         "tag_name": newVersion,
-    //         name: newVersion,
-    //         "target_commitish": "master",
-    //         body: changes
-    //     },
-    //     method: "POST",
-    //     json: true,
-    //     token: process.env.ESLINT_GITHUB_TOKEN
-    // }, function(pubErr) {
-    //     if (pubErr) {
-    //         echo("Warning: error when publishing changes to github release: " + pubErr.message);
-    //     }
-    echo("Publishing to npm");
-    exec("npm publish");
-
-    echo("Generating site");
-    target.gensite();
-    target.publishsite();
-    // });
+    ghGot("repos/eslint/eslint/releases", {
+        body: JSON.stringify({
+            "tag_name": newVersion,
+            name: newVersion,
+            "target_commitish": "master",
```

Comment:
That doesn't make sense. If it was enforcing camel-cased names, it should catch quoted ones, too. Anyway, the camelcase rule allows the configuration `{"properties": "never"}`, which is way better than relying on this hack.